x-pack/plugins/ml/public/explorer/explorer_charts/explorer_charts_container_controller.js

```diff
@@ -284,8 +284,10 @@ module.controller('MlExplorerChartsContainerController', function ($scope, $inje
       // Only plot charts for metric functions, and for detectors which don't use categorization
       // or scripted fields which can be very difficult or impossible to invert to a reverse search.
       const job = mlJobService.getJob(record.job_id);
-      if (isTimeSeriesViewDetector(job, record.detector_index) === false ||
-        _.indexOf(FUNCTION_DESCRIPTIONS_TO_PLOT, record.function_description) === -1) {
+      if (
+        isTimeSeriesViewDetector(job, record.detector_index) === false ||
+        !FUNCTION_DESCRIPTIONS_TO_PLOT.includes(record.function_description)
```

Comment:
nit, one condition uses `=== false` and one uses `!`
personally I find ` === false` to be easier to read